### DISCUSSION TITLE
Can Phoebe model tidally excited pulsations?

### DISCUSSION THREAD

**Discussion by rfreire-400** [phoebe-project/phoebe2#733] [Questions]
Hi,

I'm working on a binary system and I have evidence to believe that it emits tidally excited pulsations. Does Phoebe have any way to model that?

Thanks

**Comment by kecnry** [comment]
Not yet, unfortunately, although pulsations has had some development and is on our roadmap for a future release.

**Reply by rfreire-400** [reply]
Understood, thanks!
